app/helpers/user_journey_helper.rb

```diff
@@ -50,4 +50,12 @@ def expiry_label(certificate)
   def certificate_expiry_count(components)
     components.map(&:current_certificates).flatten.select(&:expires_soon?).count
   end
+
+  def idp_member?
```

Comment:
I think we should rename this, the reason is the we have `member` having a different meaning so this could get a little confusing
maybe change `idp_member?` to `idp_team_type` and `rp_member` to `rp_team_type` to make it really clear